sdk/cosmosdb/cosmos/src/utils/logger.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+* Cosmos Diagnostics used for all clients within the Cosmos package
+*/
+const cosmosDiagnostics = require('@azure/logger');
```

Comment:
can we use module import syntax instead?

Reply:
this is the azure logger implementation.  link here.
https://github.com/azure/azure-sdk-for-js/tree/main/sdk/core/logger

Reply:
He means write it as `import * as logger from "@azure/logger"`

Reply:
It does not.
error TS2339: Property 'log' does not exist on type 'typeof import("C:/gitrepo/azure-sdk-for-js/sdk/core/logger/types/logger")'.